### ISSUE TITLE
[Enhancement] Limit numbers of player types on roster

### ISSUE THREAD

**Issue by hutchinsfairy** [JoePym/calculateyourtv#5]
Would it be possible to have the calculator enforce numbers of player types? e.g. Not allow (or maybe just highlight) the addition of a second Minotaur or fifth Chaos Warrior to a chaos team.

I realise that it's primary purpose may be to calculate TV for an existing team but I find it a useful way to play around with new team build options.

**Comment by Kurgol** [comment]
Agreed, but if no restriction could you at least list the max number per position?
